index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Product Landing Page</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+<header id = "header">
+    <img src = "/images/company-logo.png" alt = "Company Logo" id = "header-img">
+    <nav id = "nav-bar">
+          <button class = "nav-link" href = "#Features">Features</button>
+          <button class = "nav-link" href = "#How_It_Works">How It Works</button>
+          <button class = "nav-link" href = "#Pricing">Pricing</button>
+    </nav>
+   
+   
+    
+</header>
+<div class="div-class">
+  <h1>Prototype Title</h1>
+<video id = "video" controls width = "250">
+<source src = "images/video.mp4" type="video/mp4">
+</video>
+<form id = "form" method="POST" action="https://www.freecodecamp.com/email-submit">
+    <br>
+    <br>
+    <div id = "div-input">
+<input type = "email" id = "email" placeholder="Insert e-mail here..." name = "email">
+<input id="submit" type="submit" value="submit">
+</div>
+<br>
+<br>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    
+    <script  src="js/index.js"></script>
+    <section id = "Features" class = "nav-link">
+    <h2>Features</h2>
```

Comment:
I understand your thought process behind this, but it can be improved. Your h2 tags are not linked to the navigation bar because you didn't give it an ID attribute. If you add this to the proper href in your navigation bar, it will work!